src/scss/index.scss

```diff
@@ -63,6 +63,7 @@ $color-navigation-background: $color-brand;
 @import "../components/Banner/banner";
 @import "../components/ChipGroup/chip-group";
 @import "../components/ContentReveal/content-reveal";
+@import "../components/DevBar/dev-bar";
```

Comment:
We'll be shipping this and the components in the public bundle, but I don't know if there's an easy way around that.